lib/chat_web/live/main_live/page/lobby.ex

```diff
@@ -219,6 +211,39 @@ defmodule ChatWeb.MainLive.Page.Lobby do
   defp init_new_mode(%{assigns: %{lobby_mode: :admin}} = socket),
     do: socket |> Page.AdminPanel.init()
 
+  defp process_pending_requests(%{assigns: %{room_map: room_map}, root_pid: root_pid} = socket) do
+    Task.async(fn ->
+      room_map
+      |> Map.keys()
+      |> Enum.map(fn room_hash -> {room_hash, Rooms.list_pending_requests(room_hash)} end)
+      |> Enum.each(fn {room_hash, requests} ->
+        Enum.each(requests, fn {user_hash, _} ->
+          send(root_pid, {:room_request, room_hash, user_hash})
```

Comment:
Why sending it in LiveView process, instead of handling in Task ?

Reply:
done